content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_certs_check-expiration.md

```diff
@@ -30,6 +30,13 @@ kubeadm certs check-expiration [flags]
 </colgroup>
 <tbody>
 
+<tr>
+<td colspan="2">--allow-missing-template-keys&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: true</td>
```

Comment:
this should not be a kubeadm flag, we need to understand what is going on here.
https://github.com/search?q=repo%3Akubernetes%2Fkubernetes%20allow-missing-template-keys&type=code